spec/models/api/geographical_area_spec.rb

```diff
@@ -85,6 +85,29 @@
     end
   end
 
+  describe '.non_eu_countries' do
+    let(:eu_member_states_ids) do
+      %w[AT BE BG CY CZ DE DK EE ES EU FI FR GR HR HU IE IT LT LU LV MT NL PL PT RO SE SI SK]
+    end
+    let(:members) { described_class.non_eu_countries.map(&:id) }
+
+    it 'returns non-eu results' do
+      expect(members.length).to be_positive
```

Comment:
I think this is implicitly tested on line 98 below so can be removed, too

Reply:
Good point...